examples/business_verification.ts

```diff
@@ -0,0 +1,41 @@
+// Change to smile-identity-core if you're using this snippet in your project.
+/* eslint-disable import/no-relative-packages */
+import { IDApi } from '..';
+
+// Initialize
+// Login to the Smile Identity Portal to view your partner id.
+const partner_id = '<Your partner ID>';
+// Copy your API key from the Smile Identity portal.
+const api_key = '<Your API key>';
+// Use '0' for the sandbox server, use '1' for production server.
+const sid_server = '<0 or 1>';
+
+const connection = new IDApi(partner_id, api_key, sid_server);
+
+// Create required tracking parameters
+const partner_params = {
+  job_id: '<put your unique job ID here>',
+  user_id: '<put your unique ID for the user here>',
+  job_type: 1,
```

Comment:
Can this be changed to 7 since it is business verification?
```suggestion
  job_type: 7,
```